index.js

```diff
@@ -37,19 +38,32 @@ module.exports.iMeanMost =
 module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
   function nobodyEvahExplainedItToMe() {
     return "Scientists still aren't sure, but we have a suspicion that it " +
-      "just might be a cereal involving cinnamon, toast, and something " +
-      "crunchy!"
+      "just might be something edible involving cinnamon, toast, and something " +
+      "umm... crunchy!"
   }
 
 module.exports.butWhyIsCinnamonToastCrunchGood = 
   function nobodyEvahExplainedThisToMeEither () {
-    return "It's simply a fact of life, you know? Birds fly, gravity falls, " +
-      "stevens universe, Cinnamon Toast Crunch is good."
+    return "It's simply a fact of life, you know? Birds fly, dogs walk, " +
+      "giraffes are tall, Cinnamon Toast Crunch is good."
   }
 
 module.exports.butCanIHaveItForDinner = 
-  function nobodyToldMeYouThatYouCould() {
+  function nobodyToldMeThatYouCouldnt() {
     return "Haven't heard of it, but you can have it for breakfast... and lunch... " + 
       "and as a snack... and pretty much whenever you want... " +
-      "I guess that includes dinner too."
+      "Maybe then you can have it for dinner as well!"
+  }
+
+module.exports.whichMilkCanIHaveItWith =
+  function* getMilkCombinations() {
+    yield "with 2% milk"
+    yield "with whole milk"
+    yield "with almond milk"
+    yield "with coconut milk"
+    yield "with soy milk"
+    yield "with cinnamon milk"
+    yield "with your favorite milk"
+    return "or even dry."
+    // console.log("Dry! WHAT! How dare you!")
```

Comment:
@towerofnix  Let me know if I should remove this line...

Reply:
Please:

1. Drop this commented-out line
2. As discussed in https://github.com/towerofnix/why-do-kids-love-the-taste-of-cinnamon-toast-crunch/pull/18#issuecomment-2259321556, change `return` to `yield`
3. Get rid of the period at the end, just leave it as `"or even dry"`
    * The output value of this function (milk combinations) belongs in the same part of a human sentence every time. We don't need to imply the sequence is over by using `.`—the consumer of the generator will already understand when the sequence is done. (Technically it's *not* done after this `yield`. It's still going to do a final "yield" with `{value: undefined, done: true}`, as expected!)